### PR TITLE
feat:-nav and text utilities done

### PR DESCRIPTION
**Pull request by shashikantmaurya0007**
Screenshot (1596)
Screenshot (1597)
Screenshot (1598)
Screenshot (1599)
Screenshot (1600)

### CODE REVIEW

File: Navigation/Navigation.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (zabihshaik) [inline Navigation/Navigation.html:8]: Better to add a title

File: Navigation/Navigation.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                    <a href="../Text_Utilities/text_utilities.html">Text utilities</a>
+                </li>
+                <li>
+                    <a class="active" href="../Navigation/Navigation.html">Navigation</a
+            >
+          </li>
+        </ul>
+      </aside>
+
+      <section class="right-side-of-page">
```
Comment (zabihshaik) [inline Navigation/Navigation.html:47]: Better to have class names within 3 words

**Review by zabihshaik — APPROVED** [review]
Everything looks good Just a couple of suggestions.
UI and Folder Structure is good.

**Review by manojncsoni007 — APPROVED** [review]
Everything looks good to me

**Pull request merged**